rust/tests/utils.rs

```diff
@@ -4,8 +4,14 @@ use std::path::PathBuf;
 
 use libipld::{cid::Cid, codec::Codec, ipld::Ipld, IpldCodec};
 
-static FIXTURE_SKIPLIST: [(&str, &str); 1] =
-    [("int--11959030306112471732", "integer out of int64 range")];
+static FIXTURE_SKIPLIST: [(&str, &str, &str); 1] = [
+    // This test is skipped as the current DAG-JSON decoder decodes such an integer into a float.
+    (
+        "dag-json",
```

Comment:
so this is here for the `codec=="dag-json"` right, not the name of a test?

Reply:
oh, no, I get it, this is a single skip but now it's being restricted to `"dag-json"` only, but dag-cbor hitting this will pass through and be tested